x-pack/plugins/apm/public/components/shared/links/apm/transaction_detail_link.tsx

```diff
@@ -33,6 +40,8 @@ const persistedFilters: Array<keyof APMQueryParams> = [
   'serviceVersion',
 ];
 
+const txGroupsDroppedBucketName = 'other';
```

Comment:
I think we decided to call this `_other`

```suggestion
const txGroupsDroppedBucketName = '_other';
```